site/site.go

```diff
@@ -160,6 +160,11 @@ func New(opts *Options) *Handler {
 	handler.buildInfoJSON = html.EscapeString(string(buildInfoResponse))
 	handler.handler = mux.ServeHTTP
 
+	handler.installScript, err = parseInstallScript(opts.SiteFS, opts.BuildInfo)
+	if err != nil {
+		_ = xerrors.Errorf("install.sh will be unavailable: %w", err)
```

Comment:
```suggestion
		_ = xerrors.Errorf("install.sh is unavailable: %w", err)
```

Reply:
On second thought, why do we do this? We're not logging or anything 🤔

Reply:
I wrote this as `fmt` and it changed it on me 🙄 this was supposed to be printed